go.mod

```diff
@@ -29,14 +28,12 @@ require (
 	golang.org/x/net v0.0.0-20191209160850-c0dbc17a3553 // indirect
 	golang.org/x/tools v0.0.0-20191025023517-2077df36852e // indirect
 	gopkg.in/yaml.v2 v2.2.8
-	k8s.io/api v0.17.3
-	k8s.io/apiextensions-apiserver v0.17.2
-	k8s.io/apimachinery v0.17.3
-	k8s.io/client-go v0.17.3
-	k8s.io/code-generator v0.17.3
-	sigs.k8s.io/controller-runtime v0.5.1
-	sigs.k8s.io/controller-tools v0.2.6
+	k8s.io/api v0.18.3
+	k8s.io/apiextensions-apiserver v0.18.3
+	k8s.io/apimachinery v0.18.3
+	k8s.io/client-go v0.18.3
+	k8s.io/code-generator v0.18.3
+	sigs.k8s.io/controller-runtime v0.6.0
+	sigs.k8s.io/controller-tools v0.3.0
 	sigs.k8s.io/kind v0.7.0
 )
-
-replace k8s.io/code-generator v0.17.3 => github.com/kudobuilder/code-generator v0.17.4-beta.0.0.20200316162450-cc91a9201457
```

Comment:
Yay, we can finally get rid of https://github.com/kudobuilder/code-generator !
Should probably wait with deleting it until all branches catch up with this commit.

Reply:
And with a similar commit in `kudobuilder/kudo` I guess!

Reply:
Yes, take a look at https://github.com/kudobuilder/kudo/pull/1537